### PR TITLE
feat: add authors configuration

### LINKED ISSUE #1: add author related configuration

### PR DESCRIPTION
**Pull request by maelle**
Fix #1 

Added in a bit haphazard way at the moment

### CODE REVIEW

File: inst/pkgdown/_pkgdown.yml [hunk 1]
```diff
@@ -7,3 +7,29 @@ template:
...
+  Posit Software, PBC:
+    href: https://www.posit.co
+    html: >-
+      <img src='https://www.tidyverse.org/posit-logo.svg' alt='Posit' height='16' style="margin-bottom: 3px;" />
+  Hadley Wickham:
+    href: https://hadley.nz
+  Jeroen Ooms:
+    href: https://github.com/jeroen
+  Kirill Müller:
+    href: https://cynkra.com/about/
```
Comment (krlmlr) [inline inst/pkgdown/_pkgdown.yml]:
```suggestion
    href: https://krlmlr.info/
```

**Review by krlmlr — APPROVED** [review]
@nbenn: Where would you like your "Nicolas Bennett" link from https://adbi.r-dbi.org/ to point to?

**Comment by maelle** [comment]
@nbenn friendly reminder about the question above

**Comment by nbenn** [comment]
@maelle sorry about that, I thought I had replied at some point. Didn't hit the "Comment" button I suppose.

I think my Gh profile page at https://github.com/nbenn is fine. Thanks!

**Comment by maelle** [comment]
> I thought I had replied at some point. Didn't hit the "Comment" button I suppose.

No worries, thanks!

**Pull request merged**